dashboard/test/ui/features/feedback.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Recommended blocks feedback
+
+Background:
+  Given I am on "http://learn.code.org/"
+  And I am a student
+
+Scenario: Attempt Bee Conditions 4-5 Level 1
+  Given I am on "http://learn.code.org/s/course3/stage/7/puzzle/1?noautoplay=true"
```

Comment:
You may want to copy this `.level` file and add it to `/s/allthethings`, or if the stage order changes we would need to update